src/d3d9/d3d9_device.cpp

```diff
@@ -856,14 +856,20 @@ namespace dxvk {
 
     VkExtent3D srcExtent = srcTexInfo->GetExtentMip(src->GetMipLevel());
 
+    VkExtent3D texLevelExtentBlockCount = util::computeBlockCount(srcExtent, srcFormatInfo->blockSize);
+    VkDeviceSize pitch = align(texLevelExtentBlockCount.width * uint32_t(srcFormatInfo->elementSize), 4);
+    VkExtent2D dstExtent = VkExtent2D{ uint32_t(pitch / srcFormatInfo->elementSize),
```

Comment:
store off ` uint32_t(pitch / srcFormatInfo->elementSize)`

Reply:
You mean in the texture or in a local variable? If it's the former, why? Seems a bit overkill considering we'd have to store that per mip.